cmd/manager/main.go

```diff
@@ -96,7 +100,17 @@ func main() {
 		os.Exit(1)
 	}
 
-	duration := 30 * time.Second
+	duration := func(defaultInterval time.Duration) time.Duration {
```

Comment:
Why is this wrapped in a fuction?

Reply:
My first idea was to implement this in the `vars()` section, but realised that I can't use the logging there.

You're right here it's not necessary to use a function.